test/IndexHNSW.test.js

```diff
@@ -0,0 +1,20 @@
+const { IndexHNSW, MetricType } = require('../lib');
+
+describe('IndexHNSW', () => {
+  describe('#constructor', () => {
+    it('1 arg will result in index with default neighbors & metric', () => {
+      const index = new IndexHNSW(2);
+      expect(index.getDimension()).toBe(2);
```

Comment:
Ideally we'd be able to check the value of neighbors & metrics, but indexes don't expose those props so I kept this way for consistency with faiss.